### ISSUE TITLE
По поводу вашего наставничества!) 

### ISSUE THREAD

**Issue by Liliia-s** [balesniy/yp-demo#2]
Здравствуйте!
Извините, что пишу вам в репозиторий проекта, но другого способа не нашла)
Скажите, пожалуйста, вы будете наставников на курсе JS1 в HTML Academy, который начнется завтра 26 мая?
С нетерпением жду ответа!
Заранее спасибо)

**Comment by balesniy** [comment]
добрый день, да, собираюсь.

**Comment by Liliia-s** [comment]
> добрый день, да, собираюсь.

а есть возможность вас выбрать в качестве наставника? в плане вы свободны?)
вас просто посоветовали как хорошего наставника и много положительных отзывов читала)

**Comment by balesniy** [comment]
да, у меня осталось одно место. напишу кураторам.25.05.2020, 18:43, "Liliia-s" <notifications@github.com>:

добрый день, да, собираюсь.

а есть возможность вас выбрать в качестве наставника? в плане вы свободны?)
вас просто посоветовали как хорошего наставника и много положительных отзывов читала)

**Comment by Liliia-s** [comment]
да, запишите меня пожалуйста! ура)
но очень жаль, что только одно( 
нас просто двое сестер и мы вдвоем к вам хотели)
в таком случае не могли бы вы посоветовать кого-то еще из знакомых наставников академии, которые тоже толковые и отличные, чтобы моя сестренка могла записаться к кому-то из них!?

**Comment by balesniy** [comment]
а что с вашими наставниками с верстки случилось?25.05.2020, 18:48, "Liliia-s" <notifications@github.com>:
да, запишите меня пожалуйста! ура)
но очень жаль, что только одно(
нас просто двое сестер и мы вдвоем к вам хотели)
в таком случае не могли бы вы посоветовать кого-то еще из знакомых наставников академии, которые тоже толковые и отличные, чтобы моя сестренка могла записаться к кому-то из них!?

**Comment by Liliia-s** [comment]
> а что с вашими наставниками с верстки случилось?25.05.2020, 18:48, "Liliia-s" <notifications@github.com>: да, запишите меня пожалуйста! ура) но очень жаль, что только одно( нас просто двое сестер и мы вдвоем к вам хотели) в таком случае не могли бы вы посоветовать кого-то еще из знакомых наставников академии, которые тоже толковые и отличные, чтобы моя сестренка могла записаться к кому-то из них!? —You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

с ним ничего не случилось) жив и здоров)
просто он не наставник JS-курсов)

**Comment by Liliia-s** [comment]
Алексей, так что вы решили? Запишите меня?)

**Comment by balesniy** [comment]
да, я сообщил кураторам, нужно подождать25.05.2020, 21:01, "Liliia-s" <notifications@github.com>:
Алексей, так что вы решили? Запишите меня?)

**Comment by Liliia-s** [comment]
> да, я сообщил кураторам, нужно подождать25.05.2020, 21:01, "Liliia-s" <notifications@github.com>: Алексей, так что вы решили? Запишите меня?) —You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

отлично, благодарю)
так вы можете кого-то еще, пожалуйста, посоветовать?

**Comment by balesniy** [comment]
Виктор Кан, Даниил Царев25.05.2020, 22:33, "Liliia-s" <notifications@github.com>:

да, я сообщил кураторам, нужно подождать25.05.2020, 21:01, "Liliia-s" notifications@github.com: Алексей, так что вы решили? Запишите меня?) —You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

отлично, благодарю)
так вы можете кого-то еще, пожалуйста, посоветовать?

**Comment by Liliia-s** [comment]
> Виктор Кан, Даниил Царев

огромное спасибо!
